blocks/filter.py

```diff
@@ -96,6 +96,16 @@ class object contained in the list.
     """
     def __init__(self, roles=None, bricks=None, each_role=False, name=None,
                  name_regex=None, applications=None):
+        if bricks is not None and not isinstance(bricks, Brick):
```

Comment:
Yeah, I know it's an entirely orthogonal issue, but I figured that since we're changing the code we might as well change that along with it, so if you could, please do!